packages/solid/components/FocusRing/setFocusRing.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2024 Comcast Cable Communications Management, LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { createEffect, on } from 'solid-js';
+import { activeElement } from '@lightningtv/solid';
+
+export const setFocusRing = focusRingRef =>
```

Comment:
we should add a type for this `focusRingRef` - my vote's for `SolidNode` but @chiefcll might have a better recommendation for a ref vs a node